server/models/dataSource.js

```diff
@@ -0,0 +1,8 @@
+module.exports = (sequelize, DataTypes) => {
```

Comment:
Could we stick with ES6 syntax? For example `export default function getDataSource() { ... }`.